### PR TITLE
Fix pianobar version

### PR DESCRIPTION
**Pull request by lordB8r**
updates pianobar version for latest

### CODE REVIEW

**Comment by bfontaine** [comment]
This PR needs to be rebased on top of `master`.

File: Library/Formula/jags.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+require 'formula'
```
Comment (bfontaine) [inline Library/Formula/jags.rb]: Did you intend to include this file in the PR?

**Comment by bfontaine** [comment]
You seem to have an outdated Homebrew; `pianobar` was updated to this version one year ago: https://github.com/Homebrew/homebrew/commit/52ef42d55132c2e14abc10cfe8dae56f3f10473e

Run `brew update` to update your installation.

**Comment by lordB8r** [comment]
jags wasn't supposed to go in, but i couldn't pull it out of my repo (even after git and full rm). I didn't know where it came from and git was showing me that it was added by a different user, so idk...

I did a clean clone of homebrew, and thought that that would give me an updated version, because the file i saw was outdated. I'll remove my fork and redo.

thanks!

**Pull request closed without merging**